### ISSUE TITLE
Update metalayer to latest version of linphone + build issues

### ISSUE THREAD

**Issue by RobGries** [BelledonneCommunications/meta-bc#2]
It appears that this meta layer is a few versions behind the current release build. 

This layer also does not target the latest versions of Yocto, can it be updated?

I am moving forward with this layer in my build configuration and I am hitting some issues with a couple dependencies (opus-bc - conflicts with libopus in meta-multimedia, libpolarssl - conflicts with mbedtls). I've been able to resolve those two issues with sticking to the later versions of those dependencies already provided in the build (basically just removed them from meta-bc, and modified the .bb files to depend on the versions already installed).

Now I'm failing sanity checks with the bctoolbox recipe, which I think is due to the bitbake file targeting earlier versions of yocto.

Any ideas? Are there any newer layers to use in my build?

**Comment by RobGries** [comment]
Anyone still looking at this repo?

**Comment by ljoli** [comment]
@RobGries : yes, you are allright. In Yocto, Morty is the last stable version. 
And we have only the version 3.10.2 of aug 2016.
The last tag is 3.12.0 : 
https://github.com/BelledonneCommunications/linphone/releases

In Yocto morty, this tag ( 3.12.0 ) does not compile. there are lot of errors...
